rust-symcrypt/src/errors.rs

```diff
@@ -0,0 +1,58 @@
+use symcrypt_sys;
+use std::convert::From;
+
+#[derive(Debug)]
+#[derive(PartialEq)]
+pub enum SymCryptError {

```

Comment:
It's probably worth adding `#[non_exhaustive]` to this enum. That will require people to have a catch all branch if they match on it, which means you can add new variants in the future.

Reply:
Will add `#[non_exhaustive]`